experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/Bots/LivePersonAgentBot.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json.Linq;
+
+namespace LivePersonAgentBot.Bots
+{
+    public class LivePersonAgentBot : ActivityHandler
+    {
+        private readonly BotState _conversationState;
+
+        public LivePersonAgentBot(ConversationState conversationState)
+        {
+            _conversationState = conversationState;
+        }
+
+        static Dictionary<string, string> Capitals = new Dictionary<string, string>
+        {
+            ["France"] = "Paris",
+            ["Italy"] = "Rome",
+            ["Japan"] = "Tokyo",
+            ["Poland"] = "Warsaw",
+            ["Germany"] = "Hamburg" // not really
+        };
+
+        protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
+        {
+            var conversationStateAccessors = _conversationState.CreateProperty<LoggingConversationData>(nameof(LoggingConversationData));
+            var conversationData = await conversationStateAccessors.GetAsync(turnContext, () => new LoggingConversationData());
+
+            var userText = turnContext.Activity.Text.ToLower();
```

Comment:
Suggest `ToLowerInvariant()` for better multi-culture support

```suggestion
            var userText = turnContext.Activity.Text.ToLowerInvariant();
```